.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "rules": {
+        "@definitelytyped/no-type-only-packages": "off"
```

Comment:
why do we need this at root if we are adding it to individual packages

Reply:
Oops, that is a typo in my script.

Reply:
Thanks for catching this; my grepping netted an empty string result and so that joined with the repo root in the script!